qark/qarkMain.py

```diff
@@ -34,7 +34,7 @@
 from modules import webviews
 from modules import report
 from modules import unpackAPK
-from lib.axmlparserpy import axmlprinter 
```

Comment:
Sorry that was unintentional, I will try again.  I realize now I have an emacs mode cleaning up whitespace and does other cosemtic operations (I assume the elpy-mode I was using).  I totally agree.